src/main/java/frc/robot/commands/climber/RunClimberCommand.java

```diff
@@ -10,42 +10,50 @@
 import frc.robot.subsystems.Climber;
 
 /**
- * Runs a {@link Climber}
+ * Runs the two {@link Climbers}
  */
 public class RunClimberCommand extends CommandBase {
-  private final Climber climber;
+  private final Climber leftClimber;
+  private final Climber rightClimber;
   private final Supplier<Double> speedSupplier;
 
   /**
-   * Runs the {@link Climber} at the given speed
+   * Runs the {@link Climbers} at the given speed
```

Comment:
Replace with: `Runs the {@link Climber}s at the given speed` or something similar.

I don't think the javadoc will work if you put the s inside the {}s.